lib/src/clojure_lsp/shared.clj

```diff
@@ -421,8 +421,7 @@
        result#)))
 
 (defmacro logging-task [task-id & body]
-  (with-meta `(let [msg# (str ~task-id " %s")]
-                (logging-time msg# ~@body))
+  (with-meta `(logging-time (str ~task-id " %s") ~@body)
```

Comment:
This restores file line numbers to the logs, something I accidentally broke recently.